imports/client/components/ChatPeople.tsx

```diff
@@ -156,7 +148,14 @@ const ChatPeople = (props: ChatPeopleProps) => {
 
   const loading = subscribersLoading() || callMembersLoading() || avatarsLoading();
 
-  const tracker: ChatPeopleTracker = useTracker(() => {
+  const {
```

Comment:
Yeah, I agree that this is still a risk. I think the destructuring over using a tracker object is the big win here. It's relatively easy to have fields on the `tracker` object that never get accessed - I found a few putting this commit together - and I think the indirection makes it less likely you directly compare the destructuring assignment against the data object that you're returning. But there's definitely nothing statically preventing it.

I haven't been able to find any way to automatically lint or statically check for this. One option would be to do something like https://medium.com/@_achou/using-typescript-to-statically-check-exhaustive-destructuring-7d56a7514127 anywhere we still are using destructuring assignment (instead of one-per-variable), or anywhere the `useTracker` logic is particularly complex.

Reply:
Neat!  Happy to punt on that for now, but cool to think about ways we can be stricter. :)